### ISSUE TITLE
Reject Alternate IPs per each Account Key

### ISSUE THREAD

**Issue by Forman9** [jasonrohrer/OneLife#1140]
This is an easy plug for a lot of attack surface... (like 5-10 lines?)

You'll probably think "people change ip addresses" and... "I don't want to do it manually" 

but...

That's pretty rare and you don't have the user load that this would be an issue most likely

but still...

You could easily create a main menu option to add/remove/change registered ip addresses --
-- to your own specification (and obviously log this)

My main point is I don't think /most/ account "sharers" are bad actors --
-- and the logs would obviously identify any /actual/ bad actors

NOTE: If this is implemented, I think you should do 1 hard reset for all game keys to their associated emails  --
-- it seems only fair

**Comment by MrMendeleev2** [comment]
People on dynamic IP addresses be like. _sure..._

**Comment by Forman9** [comment]
> People on dynamic IP addresses be like. _sure..._

Dynamic IPs have ranges iirc

And again, the player base is so tiny the odds of overlap are miniscule.

**Comment by jasonrohrer** [comment]
Yeah, this would be a huge pain for people on dynamic IPs, which is like 99% of the world (including me).
